identity-apps-core/apps/recovery-portal/src/main/webapp/username-recovery.jsp

```diff
@@ -376,6 +382,24 @@
                     }
                 <% } %>
 
+                // Contact input validation.
+                const contact = $("#contact").val();
+                const mobileClaimRegex = new RegExp("^\\s*(?:\\+?(\\d{1,3}))?[-. (]*(\\d{2,3})[-. )]*(\\d{3})[-. ]*(\\d{4,6})(?: *x(\\d+))?\\s*$");
```

Comment:
check if we can use java regex from framework or other repo?

Reply:
Fixed by https://github.com/wso2/identity-apps/pull/7005/commits/a92062dd0416fccb38c86f10bb10fe99a9574cb6